### DISCUSSION TITLE
关于登陆是否考虑多种实现方式？

### DISCUSSION THREAD

**Discussion by StringKe** [mamoe/mirai#873] [General]
1. 我是 ios 用户没有安卓设备。所有的账号没有能够登陆上的
2. qq实际上登陆 有短信验证码，设备锁这些，能否通过这种方式登陆呢？

**Comment by Him188** [comment]
#717

**Comment by Him188** [comment]
其他登录方式在计划中

**Comment by Him188** [comment]
另外登录与你是不是 iOS 设备无关, 参照文档 https://github.com/mamoe/mirai/blob/dev/docs/Bots.md#2-%E7%99%BB%E5%BD%95
